### PR TITLE
chore: add pwa instructions to docs

### PR DESCRIPTION
**Pull request by kilrau**
+ moved docs into a `/docs` subfolder just like we have it for boltz-backend, boltz-lnd etc.
+ does away with the uglyish README because of the Gitbook header pic that @michael1011 complained about

Preview: https://boltz.gitbook.io/beta-client-boltz-web-app/pwa

### CODE REVIEW

**Review by dni — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,27 +1,12 @@
...
-
-Clone the repository, change to the project folder and run `npm install` to install all dependencies. Then `npm run mainnet` and `npx serve dist` to bring it up.
-
-Open [http://localhost:3000](http://localhost:3000) in your browser and start swapping!
+We encourage our technical users to check the code and run the web app locally from source following [these instructions](https://docs.boltz.exchange/v/web-app/).
 
 ## ⚠️ License ⚠️
 
-Boltz Web App is licensed under [BSL 1.1](https://mariadb.com/bsl11/), a "source available" license which automatically turns into an open source license after 4 years, see [LICENSE](LICENSE) and [BSL FAQs](https://mariadb.com/bsl-faq-mariadb/) for details. If you are interested in a partnership to run the latest version please contact us at [hi@bol.tz](mailto:hi@bol.tz).
+Boltz Web App is licensed under [BSL 1.1](https://mariadb.com/bsl11/), a "source available" license which automatically turns into an open source license after 4 years, see [LICENSE](LICENSE) and [BSL FAQs](https://mariadb.com/bsl-faq-mariadb/) for details. If you are interested in a partnership to run a the latest version please contact us at [hi@bol.tz](mailto:hi@bol.tz).
```
Comment (michael1011) [inline README.md:9]: Run `a` or `the` latest version?

File: docs/pwa.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+description: >-
+  Boltz can be installed as Progressive Web App (PWA) on Android and iOS devices
+  as well as desktop computers.
+---
+
+# 📲 Install as App
+
+## Android
+
+1. Open [boltz.exchange](https://boltz.exchange) in a mobile browser like Chrome/Chromium, open the browser menu and tap "Install app":
```
Comment (michael1011) [inline docs/pwa.md:11]:
```suggestion
1. Open [boltz.exchange](https://boltz.exchange) in a mobile browser like Chrome or Chromium, open the browser menu and tap "Install app":
```

Is there a Chromium version on Android?

File: docs/pwa.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+<figure><img src=".gitbook/assets/ios-2.jpeg" alt=""><figcaption></figcaption></figure>
+
+4. Now you find Boltz as App Icon on your home screen which you can use just like any other app.
+
+## Desktop
+
+Here an example how to install Boltz as App on a Ubuntu Desktop Computer:
+
+1. Open [boltz.exchange](https://boltz.exchange) in browser like Chrome/Chromium and click the install icon that automatically appears:
```
Comment (michael1011) [inline docs/pwa.md:41]:
```suggestion
1. Open [boltz.exchange](https://boltz.exchange) in browser like Chrome or Chromium and click the install icon that automatically appears:
```

**Review by michael1011 — CHANGES_REQUESTED** [review]
Minor nits. Looks good!

And nice that the README isn't broken anymore :)

File: docs/pwa.md (hunk 2 above)
Comment (kilrau) [reply docs/pwa.md:11]: No official one, you are right. Replaced with "Vanadium" which is the standard "hardened Chromium" browser on GrapheneOS which I suspect a lot of our users will be using.

File: README.md (hunk 1 above)
Comment (kilrau) [reply README.md:9]: Nice catch, fixed.

File: docs/pwa.md (hunk 3 above)
Comment (kilrau) [reply docs/pwa.md:41]: Rephrased differently.

**Review by michael1011 — APPROVED** [review]

**Pull request merged**